circle.yml

```diff
@@ -31,23 +27,12 @@ test:
     - gvm use stable && go version
     - gvm use stable && make build:
         pwd: $BASE_STABLE
+    - gvm use stable && GO15VENDOREXPERIMENT=1 go get github.com/dgageot/docker-machine-driver-none-external
```

Comment:
I fear that having a folder in the `docker/machine` repo that contains a copy of some packages/dependencies of `docker/machine` itself would be a bit surprising. By having an external repo, we also protect ourselves from unwanted modifications during a refactoring. Could we have a repo under `docker` organization?